lib/session.js

```diff
@@ -24,6 +25,11 @@ module.exports = class Session {
     this[cookieSignerKey] = cookieSigner
     this[requestKey] = request
     this[sessionIdKey] = sessionId
+    this[encryptedSessionIdKey] = (
+      prevSession &&
+      prevSession[sessionIdKey] === sessionId &&
+      prevSession[encryptedSessionIdKey]
+    ) || cookieSigner.sign(this.sessionId)
```

Comment:
```suggestion
    this[encryptedSessionIdKey] =
      prevSession && prevSession[sessionIdKey] === sessionId
        ? prevSession[encryptedSessionIdKey]
        : cookieSigner.sign(this.sessionId);
```

right?

Reply:
Ternary operator is branching. It should be theoretically slower. I will make benchmarks to see if it has some impact.

Reply:
The ternary operator makes it 5 % slower.